src/components/ChatLog.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ChatEntry from './ChatEntry.js';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatEntryComponents = props.entries.map((entry) => {
+    return (
+      <ChatEntry
+        key={entry.id}
+        id={entry.id}
+        sender={entry.sender}
+        body={entry.body}
+        timeStamp={entry.timeStamp}
+        liked={entry.liked}
+        onUpdateLike={props.onUpdateLike}
+        onHeartClicks={props.onHeartClicks}
+        localSender={props.localSender}
+        colorLocal={props.colorLocal}
+        colorRemote={props.colorRemote}
+      />
+    );
```

Comment:
Beautiful map function and great job organizing what keys are under `props` and what are under `entry` 🎉